src/frequenz/reporting/_reporting.py

```diff
@@ -0,0 +1,99 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""A highlevel interface for the reporting API."""
+
+from collections import namedtuple
+from datetime import datetime
+
+from frequenz.client.common.metric import Metric
+from frequenz.client.reporting import ReportingApiClient
+
+CumulativeEnergy = namedtuple(
+    "CumulativeEnergy", ["start_time", "end_time", "consumption", "production"]
+)
+"""Type for cumulative energy consumption and production over a specified time."""
+
+
+# pylint: disable=too-many-arguments
```

Comment:
This will disable it for the whole file, I suggest using `disable-next` instead:
```suggestion
# pylint: disable-next=too-many-arguments
```